mlflow/server/handlers.py

```diff
@@ -261,12 +262,62 @@ def initialize_backend_stores(backend_store_uri=None, default_artifact_root=None
     except UnsupportedModelRegistryStoreURIException:
         pass
 
+def _assert_string(x):
+    assert isinstance(x, str)
+
+def _assert_int(x):
+    assert isinstance(x, int)
+
+def _assert_bool(x):
+    assert isinstance(x, bool)
+
+def _assert_float(x):
+    assert isinstance(x, float)
+
+def _assert_array(x):
+    assert isinstance(x, list)
+
+def _assert_required(x):
+    assert x is not None
+
+def _assert_less_than_or_equal(x, max):
+    assert x <= max
+
+def _assert_item_type_string(x):
+    for item in x:
+        _assert_string(item)
+
+
+def _validate_param_against_schema(schema, param, value, parsing_succeeded = False):
+    for f in schema:
+        # If the validator is a type checker and dict parsing already
+        #  succeeeded, we can safely skip re-checking types
+        #  see: https://github.com/mlflow/mlflow/pull/5458#issuecomment-1080880870
+        if f in [
+            _assert_int,
+            _assert_string,
+            _assert_bool,
+            _assert_float,
+            _assert_array,
+            _assert_item_type_string
+        ] and parsing_succeeded:
```

Comment:
Nit: Can we convert this list to a `set()` and pull it out of this method into a global variable so that it's only constructed once? That will make this check cheaper.